pages/community/ambassador-program/index.js

```diff
@@ -0,0 +1,298 @@
+    details:
+      'Speak at meetups and conferences, and we’ll help with slides, abstract submissions, and travel budget.',
+    icon: '/img/illustrations/speaking.svg',
+  },
+  {
+    title: 'Learning apps',
+    details:
+      'Build educational applications and games to teach developers about AsycnAPI and event driven architectures.',
+    icon: '/img/illustrations/learning-app.svg',
+  },
+  {
+    title: 'Build educational apps',
+    details: 'Develop applications for hands-on learning in any language.',
+    icon: '/img/illustrations/codes.svg',
+  },
+  {
+    title: 'AsyncAPI Advisor',
+    details:
+      'Collaborate with the AsyncAPI team to boost awareness of open source.',
+    icon: '/img/illustrations/advisor.svg',
+  },
+  {
+    title: 'Gather Usecases',
+    details: 'Collect data from existing AsyncAPI users ',
+    icon: '/img/illustrations/meeting.jpg',
+  },
+];
+
+
+const tokens = [
+  {
+    emoji: '📚',
+    title: 'Education',
+    details:
+      'Ambassadors are provided with a budget plan that they use for personal development tools, like books, courses, and other learning methods.',
+  },
+  {
+    emoji: '🗺️',
+    title: 'Travel',
+    details:
+      'Ambassadors are provided with Free entry to AsyncAPI conferences.',
+  },
+  {
+    emoji: '🌟',
+    title: 'Recognition',
+    details: 'Ambassadors have a Community-wide recognition.',
+  },
+  {
+    emoji: '🎁',
+    title: 'Special Swags',
+    details:
+      'Community members will recognize you as a leading voice for AsyncAPI by your exclusive AsyncAPI Ambassador swag you’ll have.',
+  },
+  {
+    emoji: '🧰',
+    title: 'Workshop Swags',
+    details: 'Ambassadors are provided with a Conference and workshop swag.',
+  }
+];
+
+function Index() {
+  const image = '/img/social/website-card.png';
+  return (
+    <GenericLayout
+      title="AsyncAPI Ambassador Program"
+      description="The home for developer communities"
+      image={image}
+      hideBanner={true}
+      wide
+    >
+      <div className="flex flex-col lg:flex-row justify-between items-center">
+        <div className="lg:w-[45%] w-full lg:text-left text-center">
+          <h1 className="font-semibold  text-3xl lg:text-5xl md:text-4xl mt-10">
+            Teachers. Champions.{' '}
+            <span className="countdown-text-gradient">Ambassadors!</span>
+          </h1>
+          <Heading
+            typeStyle="body-sm"
+            textColor="text-gray-700"
+            className="text-slate-500 mt-5"
+          >
+            Passionate about event driven architectures or message driven apis?
+            Become an AsyncAPI Ambassador and help the development community
+            build the future of APIs.
+          </Heading>
+          <div>
+            <Button
+              className="block md:inline-block focus:outline-none mt-10 text-center"
+              text="Apply to become one"
+              href="https://github.com/orgs/asyncapi/discussions"
```

Comment:
wrong link

Reply:
@derberg can you point me to the correct link?

Reply:
https://github.com/asyncapi/community/blob/master/AMBASSADOR_ORGANIZATION.md#are-you-interested-in-becoming-an-official-asyncapi-ambassador

Reply:
don't forget about changing link